process/transactionLog/process.go

```diff
@@ -98,19 +97,19 @@ func (tlp *txLogProcessor) GetLogFromCache(txHash []byte) (indexer.LogData, bool
 
 	txLog, err := tlp.GetLog(txHash)
```

Comment:
could have made the logsIndices actually a map of logData and thus, removing the slice indexing on L95

Reply:
I would leave it as it is as we want to make sure of the order

Reply:
Irrelevant: the map could have been used just to avoid double reads: one from a map and one from the slice